src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -18,7 +18,7 @@
 public class DriverDaoImpl implements DriverDao {
     @Override
     public Driver create(Driver driver) {
-        String query = "INSERT INTO drivers (name, license_number) VALUES (?, ?);";
+        String query = "INSERT INTO driver (name, license_number) VALUES (?, ?);";
```

Comment:
The table must be named in the plural
```suggestion
        String query = "INSERT INTO drivers (name, license_number) VALUES (?, ?);";
```